### DISCUSSION TITLE
How to set array as a default value

### DISCUSSION THREAD

**Discussion by kakarot-dev** [prisma/prisma#12532] [Q&A] [answered]
db is mongodb, type string array

**Comment by austincrim** [comment, marked as answer]
Hey @kakarot-dev 👋,

This isn't possible yet in Prisma. Please upvote [this issue](https://github.com/prisma/prisma/issues/8179) and let us know your use case to help us prioritize!

**Reply by kakarot-dev** [reply]
Thanks Buddy!!!

**Reply by kakarot-dev** [reply]
as for use case, its like if i wanna set some stuff which i dont wanna set while making the document
```
model test {
id String @map("_id") @db.ObjectId
tests_to_run String[] @default(["cargo", "npm", "pip", "go"])
}
```
Something like the above

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
